packages/snaps-sdk/src/types/methods/notify.ts

```diff
@@ -17,14 +33,47 @@ export enum NotificationType {
  * @property type - The type of notification to display.
  * @property message - The message to display in the notification.
  */
-export type NotifyParams = {
-  type: EnumToUnion<NotificationType>;
-  message: string;
-};
+export type NotifyParams =
+  | {
+      type: EnumToUnion<NotificationType.Native>;
+      message: string;
+    }
+  | {
+      type: EnumToUnion<NotificationType.InApp>;
+      message: string;
+    }
+  | {
+      type: EnumToUnion<NotificationType.InApp>;
+      message: string;
+      detailedView: NotificationComponent;
+      title: string;
```

Comment:
What's the difference between these props?

Reply:
1. `message` - what is displayed in the item view (and as well as in a section of the expanded view)
2. `detailedView` - the JSX content displayed 
3. `title` - the title shown in the expanded view header.